client/src/components/app/App.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
```

Comment:
it is a convention to keep `App.js` or the entry point of the react application in the `src` folder.

Reply:
Done